packages/orchestration/src/examples/swapExample.contract.js

```diff
@@ -80,7 +80,9 @@ export const start = async (zcf, privateArgs) => {
         agoric.makeAccount(),
       ]);
 
-      const tiaAddress = celestiaAccount.getAddress();
+      // ??? should this work without E()? if so, how do we do this until there's a membrane?
+      // @ts-expect-error XXX chain account types
+      const tiaAddress = E(celestiaAccount).getAddress();
```

Comment:
yes, it should work without E. `celestiaAccount` is a local object, like `orch` and `celestia`.

Reply:
totally. I can't remember what let me astray here